File: Beecrowd/TypeScript/beginner/beecrowd-1098.ts
import { readFileSync } from "node:fs";
import { EOL } from "node:os";
import { stdin } from "node:process";

let i = 0;

while (i <= 2) {
	console.log(`I=${rounds(i)} J=${rounds(i + 1)}`);
	console.log(`I=${rounds(i)} J=${rounds(i + 2)}`);
	console.log(`I=${rounds(i)} J=${rounds(i + 3)}`);
	i += 0.2;
}

function rounds(n: number): string {
	const rounded = Math.round(n * 10) / 10;
	return rounded % 1 === 0 ? rounded.toString() : rounded.toFixed(1);
}

File: Csharp/Beecrowd/src/Resolutions/Beginner/Beecrowd1098.cs
namespace BeeCrowd.src.Resolutions.Beginner;

public class Beecrowd1098
{
    public static void SequenceIJ4()
    {
        double i = 0;

        while (i <= 2)
        {
            Console.WriteLine($"I={Rounds(i)} J={Rounds(i + 1)}");
            Console.WriteLine($"I={Rounds(i)} J={Rounds(i + 2)}");
            Console.WriteLine($"I={Rounds(i)} J={Rounds(i + 3)}");
            i += 0.2;
        }
    }

    private static string Rounds(double n)
    {
        var rounded = Math.Round(n * 10) / 10;
        return rounded % 1 == 0 ? rounded.ToString() : $"{rounded:F1}";
    }
}